resource-server/src/main/java/com/okta/spring/example/ResourceServerExampleApplication.java

```diff
@@ -32,9 +32,7 @@ public static void main(String[] args) {
         @Override
         protected void configure(HttpSecurity http) throws Exception {
             http.authorizeRequests()
-                .anyRequest().authenticated()
-            .and()
-                .oauth2ResourceServer().jwt();
```

Comment:
User should be able to choose JWT (or) Opaque Token validation scenario at server startup by specifying boolean system property `-Dokta.oauth2.opaque` (true indicates opaque token introspection will be used; else the default is JWT validation).

Reply:
I still think it makes it more obvious to add this here. You can add a comment that `.opaquetoken()` would work too